apps/codeforafrica/src/components/OpportunityCardList/OpportunityCardList.js

```diff
@@ -16,7 +16,7 @@ const OpportunityCardList = React.forwardRef(function OpportunityCardList(
     <Stack spacing={{ xs: 5, md: 10 }} {...other} ref={ref}>
       {opportunities.map((opportunity) => (
         <OpportunityCard
-          key={opportunity.href || opportunity.title}
+          key={opportunity.id || opportunity.title}
```

Comment:
```suggestion
          key={opportunity.id}
```
Do you envision a situation where an opportunity will now have an id?